src/posts/tracker/changelog.ts

```diff
@@ -0,0 +1,384 @@
+export const mailchimpSubscribe = 'http://eepurl.com/hb6y4T'
+
+const title = 'Risk Tracker Changelog'
+const shortTitle = 'Changelog'
+
+interface Change {
+  date: Date
+  versionNum?: string
+  title?: string
+  spreadsheetURL?: string
+  whatsNew: string
+  instructions?: string
+}
+
+// NOTE: Do not include '/edit' at the end of the URL, because we append different paths like /copy or /template/preview depending on the use case. Do not include a slash at the end of the URL.
```

Comment:
Consider adding a test to verify this (or opening an issue to add a test)

Reply:
Do you mean a test in the test suite or a "test" in this file that runs every time the code runs and cleans the formatting if it was entered incorrectly?